[投稿機能]投稿の一覧/詳細/新規作成の機能を実装した。

taku824 opened the pull request:
## 関連issue
https://www.notion.so/13b3dee79cb680c99eefe918047d58b2?pvs=4

## 変更概要
ルートパスにアクセスされた場合、/postsへリダイレクトするようにしている。
更新と削除に関しては、別タスクとして細かく実装していくため、今回は実装していない。


## 参考
https://railsguides.jp/getting_started.html#%E8%A8%98%E4%BA%8B%E3%82%921%E4%BB%B6%E4%BD%9C%E6%88%90%E3%81%99%E3%82%8B

Commit:
[投稿機能]投稿のテーブルを追加した

https://www.notion.so/13b3dee79cb680c99eefe918047d58b2?pvs=4

最低限の投稿情報を持つテーブルを追加した

https://railsguides.jp/getting_started.html#%E8%A8%98%E4%BA%8B%E3%82%921%E4%BB%B6%E4%BD%9C%E6%88%90%E3%81%99%E3%82%8B

taku824 committed:
[投稿機能]投稿の一覧/詳細/新規作成の機能を実装した。

https://www.notion.so/13b3dee79cb680c99eefe918047d58b2?pvs=4

ルートパスにアクセスされた場合、/postsへリダイレクトするようにしている。
更新と削除に関しては、別タスクとして細かく実装していくため、今回は実装していない。

https://railsguides.jp/getting_started.html#%E8%A8%98%E4%BA%8B%E3%82%921%E4%BB%B6%E4%BD%9C%E6%88%90%E3%81%99%E3%82%8B

taku824 commented:
セルフレビュー未完了だけど、OPENしておいて、差分一緒に見るか

taku824 commented on app/controllers/posts_controller.rb:
```diff
@@ -0,0 +1,32 @@
+class PostsController < ApplicationController
+  allow_browser versions: :modern
+
+  def index
+    @posts = Post.all
+  end
```
📝 controllerファイルでやること
- データをとってきて、ビューにデータを返す
- ルーティングに対してメソッドを用意すること
- ステータスコードを返したりもする

taku824 commented on app/controllers/posts_controller.rb:
```diff
@@ -0,0 +1,32 @@
+  end
+
+  def show
+    @post = Post.find(params[:id])
```
📝 UserAccountクラスについての言及
https://github.com/taku824/yatax-samurai-tutorial/blob/66f7eff6c1ca5ad79412680ece619922e02d782e/app/models/user_account.rb#L1-L6

yooshi-me reviewed (approved):
レビューしました。
ある程度理解できました。

taku824 committed:
allow_browserの指定を親コントローラのみで行うようにした。

allow_browserについては以下を参考。
https://www.notion.so/allow_browser-14f3dee79cb680b59d18ffe98fc77404?pvs=4
